functions/http2/1.0/function.json

```diff
@@ -0,0 +1,108 @@
+{
+  "description": "An actionstep that allows for sending a HTTP(S) request with a dynamic body.",
+  "label": "HTTP(S) 2.0",
```

Comment:
HTTP(S) 2.0 is not a correct name. This indicates it cannot comply with other http versions